scripts/iam/README.md

```diff
@@ -0,0 +1,202 @@
+# GenAI IDP IAM Roles
+
+This directory contains IAM role definitions for operating GenAI IDP solution patterns with least-privilege access.
+
+## All Patterns Deployer Role
+
+The `all-patterns-deployer-role.yaml` CloudFormation template creates a least-privilege IAM role for deploying all three GenAI IDP patterns (1, 2, and 3). This role is designed for users who need to deploy and manage patterns but don't have administrator rights.
+
+### Key Features:
+- **Tag-based access control** - Users must be in the deployer group and use required tags
+- **All patterns support** - Can deploy Pattern 1 (BDA), Pattern 2 (Textract+Bedrock), and Pattern 3 (UDOP)
+- **CloudFormation compatible** - Can be used with `--role-arn` parameter
+- **Least privilege** - Only permissions needed for pattern deployment
```

Comment:
do you mean 'stack deployment (all patterns and options)'?    The role is not specific to individual patterns or Cfn parameter values, right?